python/paddle/fluid/core.py

```diff
@@ -316,6 +316,8 @@ def to_list(s):
     from .libpaddle import __set_fwd_prim_enabled
     from .libpaddle import _is_fwd_prim_enabled
     from .libpaddle import __set_all_prim_enabled
+    from .libpaddle import _is_eager_prim_enabled
+    from .libpaddle import __set_eager_prim_enabled
```

Comment:
why set use two _ ?

Reply:
in this way, no one can directly use it, out of core